attestation-agent/lib/src/lib.rs

```diff
@@ -94,20 +95,26 @@ pub trait AttestationAPIs {
 pub struct AttestationAgent {
     kbc_module_list: KbcModuleList,
     kbc_instance_map: HashMap<String, KbcInstance>,
+    config_file_path: String,
 }
 
 impl Default for AttestationAgent {
     fn default() -> Self {
-        Self::new()
+        AttestationAgent {
+            kbc_module_list: KbcModuleList::new(),
+            kbc_instance_map: HashMap::new(),
+            config_file_path: config::DEFAULT_AA_CONFIG_PATH.to_string(),
+        }
     }
 }
 
 impl AttestationAgent {
     /// Create a new instance of [AttestationAgent].
-    pub fn new() -> Self {
+    pub fn new(config_path: &str) -> Self {
```

Comment:
This function is never called due to the code. Is this intentded?

Reply:
This API is indeed not used in AA's gRPC binary, but AA library, as an independent Rust Crate, will provide support for a wider range of scenarios, so it is necessary to provide the API here.